CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# [3.0.0-beta.5](https://github.com/algolia/angular-instantsearch/compare/3.0.0-beta.4...3.0.0-beta.5) (2020-04-03)
+
+
+### Bug Fixes
+
+* **deps:** update dependency lodash to v4.17.13 [security] ([#656](https://github.com/algolia/angular-instantsearch/issues/656)) ([401a22e](https://github.com/algolia/angular-instantsearch/commit/401a22e530115e60de49101fc102c32cf344ce1a))
+
+
+### Features
+
+* **ssr:** forward options to search client ([#715](https://github.com/algolia/angular-instantsearch/issues/715)) ([8355c97](https://github.com/algolia/angular-instantsearch/commit/8355c971709b2aae7e4feef6c8ba61709fc43699))
+
+
+
+## [2.2.2](https://github.com/algolia/angular-instantsearch/compare/3.0.0-beta.0...2.2.2) (2019-04-18)
```

Comment:
Interesting broken behavior here @eunjae-lee.

Reply:
@francoischalifour That's indeed weird. I guess it's coming from conventional-changelog.